src/main/java/FirstClass.java

```diff
@@ -0,0 +1,46 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import java.util.concurrent.TimeUnit;
+
+public class FirstClass {
+    public static void main(String[] args) throws InterruptedException {
+        System.setProperty("webdriver.chrome.driver", "src/main/resources/chromedriver.exe");
+        WebDriver driver = new ChromeDriver();
+        driver.manage().timeouts().implicitlyWait(4, TimeUnit.SECONDS);
+        driver.manage().window().maximize();
+        driver.get("https://www.google.com/");
+        String searchFieldXpath = "//input[@type=\"text\"]";
+        WebElement searchElement = driver.findElement(By.xpath(searchFieldXpath));
+        searchElement.sendKeys("rozetka ua");
+        System.out.println("Search field is displayed " + searchElement.isDisplayed());
+        searchElement.submit();
+
+        WebElement searchResults = driver.findElement(By.xpath("//div[@class='yuRUbf']//a[@href='https://rozetka.com.ua/']//h3[@class='LC20lb MBeuO DKV0Md']"));
+        searchResults.click();
+        System.out.println("Current URL after click is " + driver.getCurrentUrl());
+
+        WebElement searchField = driver.findElement(By.xpath("//input[@class]"));
+        searchField.click();
+        searchField.sendKeys("карамель соленая spell с ванилью 500");
+        System.out.println("Search field is displayed " + searchField.isDisplayed());
+
+        WebElement searchButton = driver.findElement(By.xpath("//button[@class=\"button button_color_green button_size_medium search-form__submit ng-star-inserted\"]"));
+        searchButton.click();
+        System.out.println("Current URL after click is " + driver.getCurrentUrl());
+
+        WebElement chooseProduct = driver.findElement(By.xpath("//img[@src=\"https://content2.rozetka.com.ua/goods/images/big_tile/27924167.jpg\"]"));
+        System.out.println("Search product is displayed " + chooseProduct.isDisplayed());
+        System.out.println("Product name " + driver.getTitle());
+        chooseProduct.click();
+        driver.navigate().refresh();
```

Comment:
why do you need refresh?

Reply:
![image](https://user-images.githubusercontent.com/93332906/150768637-9f11dce3-f4e3-48ac-80c3-54050831fa88.png)
When you open the product page, the cursor points to the picture and it overlaps the buy button. Refreshing the page helps.